src/psyclone/tests/kernel_transformation_test.py

```diff
@@ -182,7 +182,7 @@ def test_new_kernel_file(tmpdir, monkeypatch):
     assert str(names[0]) == "continuity{0}_type".format(tag)
     # TODO #281 check compilation of code (needs Joerg's extension of
```

Comment:
As you've touched this test, please could you add the compilation testing and remove this TODO?

Reply:
Done.